services/ingestion/etl/2014tc16m4tn001/xls/src/lib/transform.js

```diff
@@ -0,0 +1,274 @@
+// @flow
+
+import crypto from 'crypto';
+import countries from 'i18n-iso-countries';
+import getCountryCode from '@eubfr/lib/location/getCountryCode';
+import type { Project } from '@eubfr/types';
+import sanitizeBudgetItem from '@eubfr/lib/budget/budgetFormatter';
+
+/**
+ * Preprocess `budget`.
+ *
+ * Input fields taken from the `record` are:
+ * - `Amount of the project (ERDF +IPA + national counterpart)`
+ * - `Co financing rate`
+ *
+ * @memberof 2014tc16m4tn001XlsTransform
+ * @param {Object} record The row received from parsed file
+ * @returns {Budget}
+ */
+
+const getBudget = record => {
+  const euContrib =
```

Comment:
or `record['ERDF'] + record['IPA Funds']`, it gives you the same result :)

Reply:
The idea here is to use the `Amount of the project (ERDF +IPA + national counterpart)` field which is used for the total, which could also be a result of the mentioned fields. I think it's good for now with the 2 fields used.